docker/jupyter/custom/jupyter_delocalize.py

```diff
@@ -231,5 +239,5 @@ def rename_file(self, old_path, new_path):
     super(WelderContentsManager, self).rename_file(old_path, new_path)
 
   def delete_file(self, path):
-    # TODO(IA-1049): Integrate Welder delete.
+    self._post_welder('delete', path)
```

Comment:
if welder delete fails, will we still delete the local file?

Reply:
No - this line with throw an exception. I have a test case to cover this situation.